cis/user.py

```diff
@@ -0,0 +1,66 @@
+"""First class object to represent a user and data about that user."""
+import logging
+from cis.settings import get_config
+
+
+logger = logging.getLogger(__name__)
+
+
+class Profile(object):
+    def __init__(self, boto_session=None, profile_data=None):
+        """
+        
+        :param boto_session: The boto session object from the constructor.
+        :param profile_data: The decrypted user profile JSON.
+        """
+        self.boto_session = boto_session
+        self.profile_data = profile_data
+        self.dynamodb_table = None
+
+    @property
+    def exists(self):
+        if self._retrieve_from_vault() is not None:
+            return True
+        else:
+            return False
+
+    def retrieve_from_vault(self):
+
+        """
+        Check if a user exist in dynamodb
+
+        :user: User's id
+        """
+
+        if self.dynamodb_table is None:
+            self._connect_dynamo_db()
+
+        user_key = {'user_id': self.profile_data.get('user_id')}
+
+        try:
+            response = self.dynamodb_table.get_item(Key=user_key)
+            self.profile_data = response
+        except Exception:
```

Comment:
This way you are hiding the actual error so we don't know and cannot debug what went wrong.